internal/provider/resource_ko_build.go

```diff
@@ -109,11 +109,11 @@ func resourceBuild() *schema.Resource {
 	}
 }
 
-type buildOpts struct {
+type buildOpts struct { //nolint: unused
```

Comment:
GitHub won't let me just make it a suggestion, but it looks like we can just delete `buildOpts` and `publishOpts` entirely. I'm not sure where they came from, but `buildOptions` and `publishOptions` are the only ones that are actually used. Yay linting!

Reply:
done